label_studio/ml_models/api.py

```diff
@@ -74,3 +78,59 @@ def perform_create(self, serializer):
         serializer.validated_data['organization'] = self.request.user.active_organization
         serializer.validated_data['created_by'] = self.request.user
         serializer.save()
+
+@method_decorator(
+    name='get',
+    decorator=swagger_auto_schema(
+        tags=['Models'],
+        operation_summary='List projects compatible with model',
+        operation_description="""
+            Retrieve a list of compatible project for a specific model. For example, use the following cURL command:
+            ```bash
+            curl -X GET {}/api/models/{{id}}/compatible-projects -H 'Authorization: Token abc123'
+            ```
+        """.format(
+            settings.HOSTNAME or 'https://localhost:8080'
+        ),
+        manual_parameters=[
+            openapi.Parameter(
+                name='id',
+                type=openapi.TYPE_INTEGER,
+                in_=openapi.IN_PATH,
+                description='A unique integer value identifying model.',
+            ),
+        ]
+    ),
+)
+class ModelCompatibleProjects(generics.RetrieveAPIView):
+
+    permission_required = all_permissions.projects_view
+ 
+    def _is_project_compatible(self, project):
+        parced_config = project.get_parsed_config()
+        if parced_config:
+            for tag in parced_config:
+                if parced_config[tag].get('type',None) == "Choices":
+                    for input in parced_config[tag].get('inputs',[]):
+                        if input.get('type', '') == 'Text' and input.get('value','') == 'text':
```

Comment:
I think the value here is just the name of the column, which could be anything, so dont need to check for value just type

Reply:
ah got it. I will remove the check